src/mpl/configure.ac

```diff
@@ -1405,6 +1406,9 @@ AX_GCC_VAR_ATTRIBUTE(used)
 
 PAC_POP_ALL_FLAGS
 
+MPL_CFLAGS="${CFLAGS#${PARENT_CFLAGS}}"
```

Comment:
Could you add a short comment explaining what the substitution does?

Reply:
This is supported in POSIX. See https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html, section 2.6.2 Parameter Expansion
```${parameter#[word]}
Remove Smallest Prefix Pattern. The word shall be expanded to produce a pattern. The parameter expansion shall then result in parameter, with the smallest portion of the prefix matched by the pattern deleted. If present, word shall not begin with an unquoted '#'.```